app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/data/ChatRepositoryImpl.kt

```diff
@@ -76,6 +76,7 @@ internal class ChatRepositoryImpl(
         messages = result.chat.messages.mapNotNull { it.toChatMessage() },
         nextUntil = result.chat.nextUntil,
         hasNext = result.chat.hasNext,
+        informationMessage = null, // TODO query from schema
```

Comment:
Will backend return this message all the time? Is it something that is not dismissible in general? I suppose it's only to be used in situations where something is wrong right? Like a downtime, that the chat is closed at this time of day etc.?